docs/run/primitives-examples.mdx

```diff
@@ -392,13 +496,20 @@ Use Sampler to return the measurement outcome as a quasi-probability distributio
 import numpy as np
 from qiskit.circuit.library import IQP
 from qiskit.quantum_info import random_hermitian
-from qiskit_ibm_runtime import QiskitRuntimeService, SamplerV2 as Sampler
- 
-service = QiskitRuntimeService()
- 
-backend = service.least_busy(operational=True, simulator=False, min_num_qubits=127)
- 
-n_qubits = 127
+from qiskit_ibm_runtime import SamplerV2 as Sampler
+from qiskit_ibm_runtime.fake_provider import FakeSherbrooke
+
+# Run the sampler job locally using FakeSherbrooke
+backend = FakeSherbrooke()
+
+# Use the commented code to run on a real system:
+
+# from qiskit_ibm_runtime import QiskitRuntimeService
+# service = QiskitRuntimeService()
+# backend = service.least_busy(operational=True, simulator=False, min_num_qubits=127)
+
+# n_qubits = 127
+n_qubits = 12
```

Comment:
Have you considered reducing the qubit count even more In the snippets that show counts? given that we are already not showing "utility-scale" experiments, I don't see much of a conceptual difference between 4 and 12 qubits, but the size of the output gets considerably more manageable (and doesn't force users to scroll through endless counts dictionaries).